api/integrations/github/views.py

```diff
@@ -71,6 +71,23 @@ def create(self, request, *args, **kwargs):
             if re.search(r"Key \(organisation_id\)=\(\d+\) already exists", str(e)):
                 raise DuplicateGitHubIntegration
 
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        installation_id = instance.installation_id
+        token = generate_jwt_token(settings.GITHUB_APP_ID)
+
+        url = f"{GITHUB_API_URL}app/installations/{installation_id}"
+
+        headers = {
+            "X-GitHub-Api-Version": GITHUB_API_VERSION,
+            "Accept": "application/vnd.github.v3+json",
+            "Authorization": f"Bearer {token}",
+        }
+
+        response = requests.delete(url, headers=headers, timeout=10)
```

Comment:
nit: This is okay, but I'd very much prefer to have a dedicated interface in the `github.github` module for this, rather than having all this code right here in the viewset.

Reply:
Good catch. It is not OK, it should go into the business logic layer. I am moving all this kind of business logic out from the views and into the github.client or github.github modules.